.github/workflows/pypi_release.yml

```diff
@@ -3,7 +3,9 @@ name: Project release on PyPi
 on:
   push:
     tags:
-      - "v[0-9].[0-9]+.[0-9]+*"
+      - "v[0-9]+.[0-9]+.[0-9]+*"
```

Comment:
Small change to support major versions greater than `9`. I just hope we manage to reach that. 🤞